### DISCUSSION TITLE
August 31, 2022

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#287] [General]
# Standup style

**Comment by pcsmgnt** [comment]
Headband Jesse on the decks

**Comment by pcsmgnt** [comment]
Last: Completed team deck slide and circulated for feedback

This: First stab at TOM and Funnel

**Comment by mrmetaverse** [comment]
Last: broke down final tickets into a milestone: https://github.com/AngellXR/incubator/milestone/4, met with Seema, and Shirley 

This Next: I am going to finish the deck. #66 

Blockers: the deck needs to be done, and a link back to angellxr needs to be added to the fund one pager

**Comment by mrmetaverse** [comment]
Seema: 

Introduced herself, 

Y: Met with Jesse to walk through the strategy, and is working on planning for DC startup week. 

T: Will continue to collab with Jesse, meeting with a lawyer to ask questions like B-corp or not 

Blockers: DC startup week planning

**Comment by mrmetaverse** [comment]
Team likes the idea of keeping this stealth for now, not viral marketing for the launch.

**Comment by mrmetaverse** [comment]
Can we collect a list of Builders, Founders, Investors, Mentors, etc.

**Comment by mrmetaverse** [comment]
Jam out on Friday for the deck - Julius

**Comment by mrmetaverse** [comment]
WE ARE DE-RISKING THE SPACE FOR FOUNDERS AND INVESTORS
